### ISSUE TITLE
Unable to install using Carthage 

### ISSUE THREAD

**Issue by bre7** [facebook/FBNotifications#11]
Is Carthage compatibility working ? I keep getting the following error message:
>A shell task (/usr/bin/env git checkout --quiet 6e7d0b36f73d960cea81f4f207010dc584bffb3b) failed with exit code 128:
fatal: Cannot switch branch to a non-commit '6e7d0b36f73d960cea81f4f207010dc584bffb3b'

I've read similar issues (related to Carthage & submodules), but couldn't find a solution.

Related issues:
```
https://github.com/Carthage/Carthage/issues/936#issuecomment-159097500
https://github.com/Carthage/Carthage/issues/135
```

Steps:

1. Create a file named `Cartfile`
2. Add `github "facebook/FBNotifications"` to it
3. Run `carthage update`

**Comment by nlutsenko** [comment]
Yup, looks like our submodule is not properly set up.
We have tests for Carthage, but looks like those are not helpful, since they aren't testing remote installation flow.

Am going to submit a fix for this in a few moments and we'll ship it this week.
Thanks for reporting!

**Comment by bre7** [comment]
Thanks for the quick response & fix :wink: Can't wait to get beta access.

**Comment by nlutsenko** [comment]
Hey @bre7, I've just shipped `ios-1.0.1` that includes this fix.
After a tad of testing with a Cartfile that contains `github "facebook/FBNotifications"` everything seems to work like a charm now.
Thank you again for reporting this! And let me know if you run into any other issues.
